src/Driver/Driver.jl

```diff
@@ -340,10 +335,24 @@ function init(;
     init_driver::Bool = true,
     keyword_args...,
 )
+    # `Settings` contains global defaults
+    global_defaults = Dict{Symbol, Any}(
+        (n, getproperty(Settings, n)) for n in propertynames(Settings)
+    )
+
+    # keyword arguments must be applicable to `Settings`
     all_args = Dict{Symbol, Any}(keyword_args)
+    for kwarg in keys(all_args)
+        if get(global_defaults, kwarg, nothing) === nothing
+            @warn("Unrecognized keyword argument '$(string(kwarg))'; ignoring")
```

Comment:
Are there any disadvantages to making this return an error instead of a warning?